File: src/main/java/ru/herobrine1st/fusion/util/VkApiUtil.java
package ru.herobrine1st.fusion.util;

import okhttp3.HttpUrl;
import ru.herobrine1st.fusion.Config;

import java.util.Objects;

public class VkApiUtil {
    public static HttpUrl.Builder getHttpUrlBuilder(String method) {
        return Objects.requireNonNull(HttpUrl.parse("https://api.vk.com/method/" + method)).newBuilder()
                .addQueryParameter("access_token", Config.getVkServiceToken())
                .addQueryParameter("v", Config.getVkAPIVersion());
    }
}

File: src/main/java/ru/herobrine1st/fusion/util/VkApiUtil.kt
package ru.herobrine1st.fusion.util

import okhttp3.HttpUrl
import okhttp3.HttpUrl.Companion.toHttpUrl
import ru.herobrine1st.fusion.Config
import java.util.*

object VkApiUtil {
    fun getHttpUrlBuilder(method: String): HttpUrl.Builder {
        return Objects.requireNonNull("https://api.vk.com/method/$method".toHttpUrl()).newBuilder()
            .addQueryParameter("access_token", Config.vkServiceToken)
            .addQueryParameter("v", Config.vkAPIVersion)
    }
}